phy/mod_difest.F

```diff
@@ -283,6 +285,15 @@ subroutine init_difest
 c    .          lnoDGat1=.true.                 ,
 c    .          CVMix_kpp_params_user=KPP_params )
 c
+c$OMP PARALLEL DO PRIVATE(l,i)
+      do j=1,jj
+        do l=1,isp(j)
+        do i=max(1,ifp(j,l)),min(ii,ilp(j,l))
+          OBLdepth(i,j)=10.
```

Comment:
OBLdepth is passed to the coupler with NUOPC, while it was not with MCT. OBLdepth is needed in case the wave component is active. The export coupling fields have to be sent before the model has started integrating, so before difest_vertical* is called. Therefore it is necessary to give OBLdepth a sensible value in the case vcoord_type_tag == cntiso_hybrid .